docs/src/main/asciidoc/security-openid-connect-concept.adoc

```diff
@@ -3,21 +3,24 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Using OpenID Connect (OIDC) to Protect Service Applications using Bearer Token Authorization
-include::_attributes.adoc[]
+[id="security-oidc-bearer-authentication-concept"]
+= OIDC Bearer authentication
+include::./attributes.adoc[]
 :categories: security
-:summary: This guide demonstrates how your Quarkus application can use Keycloak to protect your JAX-RS applications using bearer token authorization, where these tokens are issued by a Keycloak server.
+:toc:
 
-You can use the Quarkus OpenID Connect (OIDC) extension to secure your JAX-RS applications using Bearer Token Authorization. 
-The Bearer Tokens are issued by OIDC and OAuth 2.0 compliant authorization servers, such as https://www.keycloak.org[Keycloak]. 
+== Overview
```

Comment:
Would it help findability if we add more keywords to this title?

Reply:
Updated. Thanks.